api/host_test.go

```diff
@@ -65,7 +65,7 @@ func TestIntegrationHosting(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	expRevenue := "15307662222190387473"
+	expRevenue := "15307662222222127617"
```

Comment:
it's mentioned in the commit message for human readable conversions. The conversion that's happening now has a higher precision (does all the multiplies first, then the divides second) than previously (do a mult, then div, then mult), which resulted in the value changing slightly.